apps/examples/wifi_manager/wm_test/Makefile

```diff
@@ -36,8 +38,15 @@ wm_test_stress_test1.c\
 wm_test_stress_test2.c\
 wm_test_stress_test3.c\
 wm_test_stress_test4.c\
+wm_test_stress_test5.c\
+wm_test_stress_test5_client.c\
+wm_test_stress_test5_tls_server.c\
```

Comment:
Those TLS files have dependency to mbedTLS. so you'd better to add config comparison.

Reply:
Changes done as suggested.